frontend/src/metabase-lib/v1/expressions/diagnostics.ts

```diff
@@ -60,7 +60,7 @@ export function diagnose({
   stageIndex: number;
   name?: string | null;
   metadata?: Metadata;
-  expressionPosition?: number;
+  expressionPosition: number | undefined;
```

Comment:
`expressionIndex` would be a more appropriate name, given that we use `stageIndex`.

It's not great that in MLv2 they're called `stage-number` and `expression-position`, but at least we can make it consistent on FE side.